ConfigurationSystem/Client/ConfigurationClient.py

```diff
@@ -0,0 +1,85 @@
+"""
+  Custom TornadoClient for Configuration System
+  Used like a normal client, should be instanciated if and only if we use the configuration service
```

Comment:
```suggestion
  Used like a normal client, should be instantiated if and only if we use the tornado configuration service
```
Should this be with the Tornado configuration service, right?

Reply:
No, the comment is plain wrong, I'll remove it altogether.